src/main.py

```diff
@@ -208,8 +209,22 @@ def twoPlayerGame(board):
 board = Board()
 
 def main():
+    parser = argparse.ArgumentParser(prog="chess", 
+    description="A python program to play chess against an AI in the terminal.",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    epilog="Enjoy the game!"
+    )
+    parser.add_argument('-t', '--two', action='store_true', default=False, help="to play a 2-player game")
+    parser.add_argument('-u', '--unicode', action='store_true', default=False, help="display chess pieces using unicode characters")
+    parser.add_argument('-w', '--white', action='store', default="blue", metavar='W', help="color for white player")
+    parser.add_argument('-b', '--black', action='store', default="red", metavar='B', help="color for black player")
```

Comment:
All of these lines are too long. hanging indent and placing the closing bracket on a separate line and aligning it with the first character of the line that starts the construct is preferred.